internal/broker/testdata/aws-schema-additional-params-eu.json

```diff
@@ -48,6 +49,84 @@
       ],
       "type": "string"
     },
+    "modules": {
+      "_controlsOrder": [
+        "default",
+        "list"
+      ],
+      "description": "Use default modules or provide your custom list of modules.",
+      "oneOf": [
+        {
+          "additionalProperties": false,
+          "description": "Default modules",
+          "properties": {
+            "default": {
+              "default": true,
+              "description": "Check the default modules at: https://help.sap.com/docs/btp/sap-business-technology-platform/kyma-modules?version=Cloud.",
+              "readOnly": true,
+              "type": "boolean"
+            }
+          },
+          "title": "Default",
+          "type": "object"
+        },
+        {
+          "additionalProperties": false,
+          "description": "Define custom module list",
+          "properties": {
+            "list": {
+              "description": "Select a module technical name from the list available at: https://help.sap.com/docs/btp/sap-business-technology-platform/kyma-modules?version=Cloud. You can only use a modules technical name once.",
```

Comment:
`You can only use a modules technical name once` - why `modules`.
And this is kind of vague. Do you mean that used module names shall be unique?
 This comment applies to many copies of this schema part.